lib/generator/exercise_cases.rb

```diff
@@ -25,6 +25,23 @@ def indent_lines(code, depth, separator = "\n")
     code.join(separator + ' ' * depth)
   end
 
+  # use to indent multi line workloads with blank
+  # lines
+  #   indent_text(4, lines.join("\n"))
+  def indent_text(depth, text)
+    text.lines.reduce do |obj, line|
+      obj << (line == "\n" ? line : ' ' * depth + line)
+    end
+  end
```

Comment:
I wrote the methods in exercise_cases as I worked my way through moving the template files. 

I could artificially go back and separate out the two, but it feels, well, artificial?